book/src/1.welcome/README.md

```diff
@@ -1,54 +1,19 @@
-# 📖 Teams SDK: Typescript: Documentation
+# 📖 Teams SDK v2
 
-Microsoft Teams is a feature rich messaging platform. It provides features that developers can use
-to create immersive app experiences, below are some of the different packages/features that can be used with this SDK.
+Teams SDK v2 is a suite of packages used to build on Microsoft Teams Platform.
 
-## [`@microsoft/teams.apps`](https://www.npmjs.com/package/@microsoft/teams.apps)
+## ⭐ What's new?
 
-Tools for building server side apps for Microsoft Teams.
+### Streamlined Developer Experience
+We’ve refined the developer experience so you can concentrate on building your app’s logic— not wrestling with integrations.
 
-## [`@microsoft/teams.api`](https://www.npmjs.com/package/@microsoft/teams.api)
+### Effortless Integration
+We’ve simplified complex integration workflows to help you deliver a richer, more seamless user experience.
 
-Core types and client implementations for the Teams Platform API,
-used for fetching data, authentication, and sending activities.
+### Accelerate Your Workflow
+Get your application up and running in under 30 seconds with our lightning-fast CLI—so you can spend more time on what really matters.
 
-## [`@microsoft/teams.ai`](https://www.npmjs.com/package/@microsoft/teams.ai)
+## 🔎 Navigation Tips
```

Comment:
Should we include CTA? Where to file issues, ask questions etc? Maybe we need to figure that out first...

Reply:
I would expect to find 'github related' info on the readme of the repo, not inside these docs. Previously I had Contributing docs in mdbook, but Alex pointed out it makes more sense to live on the repo. I think the same makes sense for issues.

For filing issues/asking questions, I think we decided we DON'T want to keep the Discussions section open going forward, so 'dev questions' would either be a type of issue to file and/or link to the Teams channel for support. 

My 2 cents.

----

Do we want something like MSDocs has where users can file an issue directly to one of our docs files if they notice a typo or discrepancy?

Reply:
I agree with keeping it on the Github repo, direct filings would be nice as well